lib/api/route/streams.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../core.dart';
+part 'streams.g.dart';
+
+/// https://zulip.com/api/get-stream-topics
+Future<GetTopicsResult> getStreamTopics(
```

Comment:
OTOH I guess keep the method name the same as the operation is called in the API docs, like this.